modules/fee/handler_test.go

```diff
@@ -0,0 +1,134 @@
+package fee_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/modules/coin"
+	"github.com/tendermint/basecoin/modules/fee"
+	"github.com/tendermint/basecoin/stack"
+	"github.com/tendermint/basecoin/state"
+	"github.com/tendermint/tmlibs/log"
```

Comment:
stylistic, the standard I've seen across our libraries is separate imports with line breaks for:
- standard libs
- non-tendermint libs
- tendermint libs of the same package
- tendermint libs of different packages 

This would mean we should probably have `tmlibs/log` seperated with a line break here